docs/get-started/lotus/installation.md

```diff
@@ -38,32 +38,72 @@ Building Lotus requires some system dependencies, usually provided by your distr
 | OpenSUSE           | `sudo zypper in gcc git jq make libOpenCL1 opencl-headers ocl-icd-devel clang llvm && sudo ln -s /usr/lib64/libOpenCL.so.1 /usr/lib64/libOpenCL.so`                                           |
 | Amazon Linux 2     | `sudo yum install -y https://dl.fedoraproject.org/pub/epel/epest-7.noarch.rpm; sudo yum install -y git gcc bzr jq pkgconfig clang llvm mesa-libGL-devel opencl-headers ocl-icd ocl-icd-devel` |
 
 #### Rustup
 
-Lotus needs [rustup](https://rustup.rs). The easiest way to install it is:
+Lotus needs [rustup](https://rustup.rs).
 
-```sh
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-```
+1. Download and run the installer:
 
-:::tip
-Make sure your `$PATH` variable is correctly configured after the rustup installation so that `cargo` and `rustc` are found in their rustup-configured locations.
-:::
+   ```bash
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+   ```
+
+1. Select the `default` installation option:
+
+   ```bash
+   > 1) Proceed with installation (default)
+   > 2) Customize installation
+   > 3) Cancel installation
+
+   1
+   ```
+
+1. Check that Rust is installed properly by checking the version:
+
+   ```bash
+   rustc --version
+
+   > rustc 1.46.0 (04488afe3 2020-08-24)
+   ```
+
+   You may need to log in and log out for Rust to install fully. This installer automatically adds `cargo`, `rustc` and `rustup` to your `PATH`. If you are still running into issues after logging out and back in, check the [official Rust documentation](https://www.rust-lang.org/tools/install) for support.
+
+1. With Rust installed, you can continue with the rest of the installation.
 
 #### Go
 
-To build Lotus, you need a working installation of [Go 1.14 or higher](https://golang.org/dl/):
+To build Lotus, you need a working installation of [Go 1.14 or higher](https://golang.org/dl/).
 
-```sh
-wget -c https://dl.google.com/go/go1.14.7.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
-```
+1. Download and install Go:
 
-Make sure that `/usr/local/go/bin` is in your `PATH`. If you are running into problems, check the [official Go installation instructions](https://golang.org/doc/install) for your operating system.
+   ```sh
+   wget -c https://dl.google.com/go/go1.14.7.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
+   ```
+
+1. Check that Go installed correctly:
+
+   ```bash
+   $ go version
+
+   go version go1.14.7 linux/amd64
+   ```
+
+   :::tip
+   If you get an error saying that Go is not installed, you may just need to add `/usr/local/go/bin` to your `PATH` variable:
+
+   ```bash
+   echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bashrc && source ~/.bashrc
+   ```
+
+   Once you've done that try running `go version` again. If you're still facing issues, head over to the [official Go documnetation](https://golang.org/doc/install) for further support.
+   :::
+
+1. With Go installed, you can move onto building and installing Lotus.
```

Comment:
The point of checking if Go is installed is so that the user knows if they have Go installed. But again, I have no strong feeling one way or the other on this. If you think that checking that Go is installed is completely redundant, then we can take it out.

Reply:
In the interest of time, I just removed this section.